killrvideo-commons/src/main/java/com/killrvideo/conf/KafkaConfiguration.java

```diff
@@ -8,49 +8,52 @@
 import static org.apache.kafka.clients.producer.ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG;
 import static org.apache.kafka.clients.producer.ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG;
 
+import java.util.Arrays;
+import java.util.Optional;
 import java.util.Properties;
+import java.util.stream.Collectors;
 
 import org.apache.kafka.clients.consumer.KafkaConsumer;
 import org.apache.kafka.clients.producer.KafkaProducer;
 import org.apache.kafka.common.serialization.ByteArrayDeserializer;
 import org.apache.kafka.common.serialization.ByteArraySerializer;
 import org.apache.kafka.common.serialization.StringDeserializer;
 import org.apache.kafka.common.serialization.StringSerializer;
-import org.springframework.beans.factory.annotation.Autowired;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 
-import com.killrvideo.conf.KillrVideoConfiguration;
-import com.killrvideo.discovery.ServiceDiscoveryDao;
-
 /**
  * Use Kafka to exchange messages between services. 
  *
- * @author Cedrick LUNVEN (@clunven) *
+ * @author Cedrick LUNVEN (@clunven)
  */
 @Configuration
-@Profile(KillrVideoConfiguration.PROFILE_MESSAGING_KAFKA)
+@Profile("messaging_kafka")
 public class KafkaConfiguration {
     
-    /** Name of service in ETCD. */
-    public static final String SERVICE_KAFKA = "kafka";
-    
-    /** Default CQL listening port. */
-    public static final int DEFAULT_PORT = 8082;
+    /** Initialize dedicated connection to ETCD system. */
```

Comment:
This is a nitpik, but are we still using ETCD here at all? I thought Kafka replaced it?

Reply:
We should not use ETCD anymore. ETCD use to store the list of endpoints for the backend. Kafka is used for inter-services communication. As of now Kafka is disabled and not deploy into production.

Reply:
The comment was wrong indeed